src/lib/addXP.ts

```diff
@@ -139,7 +139,7 @@ export async function addXP(userID: string, params: AddXpParams): Promise<string
 		str += params.minimal
 			? `+${Math.ceil(preMax).toLocaleString()} ${skillEmoji[params.skillName]}`
 			: `You received ${Math.ceil(preMax).toLocaleString()} ${skillEmoji[params.skillName]} XP`;
-		if (params.duration && !params.minimal) {
+		if (params.duration) {
```

Comment:
This is actually intended not to give XP/hr if minimal is selected. 

Unless Magna said it's okay to change this, I would leave it alone, and just add `minimal = false` to the `addXp()` calls where you want to see the XP/hr.

If you already got the OK from Magna, then ignore this.

Reply:
I like the minimal messages at some places but excluding the xp/hr part  I don't see necessary anywhere. It's good if xp/hr is always visible on any trip because it makes it way easier for the users to spot methods and stuff that result in a too low/too high xp/hr. For OSB especially where the xp/hrs are known but also useful for BSO. And yes most users could convert but our brains are lazy.